src/molecules/Modal/ModalFooter.native.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import View from '../../atoms/View';
+import Divider from '../../atoms/Divider';
+
+const ModalFooter = ({ children, type }) => {
+  if (type === 'centered') {
+    return (
+      <Space padding={[2, 3]}>
+        <View>{children}</View>
+      </Space>
+    );
+  }
+  if (type === 'fullscreen') {
+    return null;
+  }
+  if (type === 'bottomsheet') {
+    return (
+      <React.Fragment>
+        <Divider color="shade.920" />
```

Comment:
Confirm once with Nishant about this. Not sure if this divider should exist in all the bottomsheet